src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
The inventory for sure won't be assigned anymore in this method. The object of the player may change but this change will never reach the method.